cssom-view/Overview.bs

```diff
@@ -1082,8 +1082,7 @@ dictionary ScrollIntoViewOptions : ScrollOptions {
 partial interface Element {
   sequence&lt;DOMRect> getClientRects();
   [NewObject] DOMRect getBoundingClientRect();
-  void scrollIntoView();
-  void scrollIntoView((boolean or object) arg);
+  void scrollIntoView(optional (boolean or ScrollIntoViewOptions) arg = true);
```

Comment:
I don't think you need to define a default here. I had a similar situation with `addEventListener`. Basically if no argument is given you'll end up getting a dictionary passed to your algorithm and that dictionary should have the appropriate defaults already. Does that make sense?